tutorials/build_system/prerequisites.md

```diff
@@ -2,122 +2,201 @@
+It guides you through the Gradle project creation process with four screens.
 
-### 1.2 Create a Plugin Project from Scratch
-
-IntelliJ IDEA supports automatically creating new plugin projects using Gradle, with all the necessary `build.gradle` setup performed automatically. 
-This can also be used to convert an existing plugin to Gradle, if Gradle is not able to convert the existing project - in this case, you need to copy over the sources to the new project. 
-
-To do so, create a new project in IntelliJ IDEA by opening **File \| New... \| Project**, and select Gradle from the dialog box. 
-In the "Additional Libraries and Frameworks" page, check "IntelliJ Platform Plugin".
+### New Project Configuration Screen
+On the first screen the type of project is configured:
+* From the _product category_ pane on the left, choose _Gradle_.
+* Specify the _Project SDK_.
+  Note that an [_IntelliJ Platform Plugin SDK_](/basics/getting_started/setting_up_environment.md#configuring-intellij-platform-sdk) can be used but is not required.
+  However, select a _Project SDK_ based on the 1.8 JDK because it:
+  * Will be the default JRE used to run Gradle (this can be changed later),
+  * Will also be the Java version to use when compiling Java source.
+    As of this writing, Java 1.8 is the most recent JDK supported by the IntelliJ Platform.
+* In the _Additional Libraries and Frameworks_ panel, select _Java_ and _IntelliJ Platform Plugin_, then click _Next_.
 
 ![Select the Gradle facet in the Project Creation Wizard](img/step1_new_gradle_project.png){:width="800px"}
 
-The Project Creation Wizard will now guide you through the Gradle project creation process. 
-You will need to specify a Group ID, Artifact ID, and Version:
-
-![Specify the Group, Artifact, and Version IDs](img/step2_group_artifact_version.png){:width="800px"}
-
-It’s recommended to select the `Use default gradle wrapper` option, that way IntelliJ IDEA will install everything you need to run Gradle tasks itself. 
-
-Finally, specify a JVM Gradle will use, it can be the Project JDK. 
-You also configure this path once the project is created via **Settings \| Build, Execution, Deployment \| Build Tools \| Gradle**. 
+### Project Naming Screen
+On this, the second screen of the Wizard, specify a [Group ID, Artifact ID, and plugin Version](https://www.jetbrains.com/help/idea/gradle.html#project_create_gradle) using [Maven naming](https://maven.apache.org/guides/mini/guide-naming-conventions.html) conventions.
+* _Group ID_ is typically a Java package name, and it is used for the `project.group` value in the project's `build.gradle` file.
+* _Artifact ID_ is the default name of the project JAR file (without version).
+  It is also used for the `rootProject.name` value in the project's `settings.gradle` file.
+* _Version_ is used for the `project.version` value in the `build.gradle` file. 
+
+Click _Next_ to continue. 
+
+### Gradle Settings Screen
```

Comment:
I wouldn't dig into details here, they aren't specific to plugin development. Also next versions of IDEA will have fewer options here. I think it would be enough to say that user may use the default settings here.

Reply:
That would simplify things. I will change to say use the defaults with a link to the Gradle Settings (Help) reference page.